src/app/organizations/organization-details/organization-details.component.ts

```diff
@@ -126,4 +127,25 @@ export class OrganizationDetailsComponent implements OnInit {
                 displayBackendError(err);
             }, () => SpinnerUtil.hideSpinner());
     }
+
+    deleteMember(orgID: string, memberID: string) {
+        SpinnerUtil.showSpinner();
+        this.organizationService.removeMemberFromOrganization(orgID, memberID)
+            .subscribe(() => {
+                    SpinnerUtil.hideSpinner();
+                    swal({
+                        title: '',
+                        text: 'Success!',
+                        type: 'success'
+                    }).then(function () {
+                        this.reloadPage('/dash/manage_groups/' + this.organization.uuid);
+                    }.bind(this));
+                },
+                hideSpinnerAndDisplayError);
+    }
+
+    reloadPage(uri: string) {
+        this.router.navigateByUrl('/', {skipLocationChange: true}).then(() =>
+            this.router.navigate([uri]));
```

Comment:
Even though, this reloadPage is unnecessary in this case, why you do this promise chain with `navigateByUrl` and `navigate` chain?